extension/parquet/parquet_metadata.cpp

```diff
@@ -6,16 +6,15 @@
 
 #include "duckdb/common/multi_file/multi_file_reader.hpp"
 #include "duckdb/common/types/blob.hpp"
-#include "duckdb/common/types/column/column_data_collection.hpp"
 #include "duckdb/planner/filter/constant_filter.hpp"
 #include "duckdb/main/config.hpp"
+#include "duckdb/common/multi_file/multi_file_list.hpp"
 
 namespace duckdb {
 
 struct ParquetMetaDataBindData : public TableFunctionData {
+	vector<string> file_paths;
```

Comment:
We should really use the `OpenFileInfo` here instead of a string as well - that can contain information that can make metadata reading faster (e.g. parquet metadata size). I suspect that moving back to the multi file list here would also resolve this though.